testing/stable_sort_by_key_large.cu

```diff
@@ -120,8 +120,8 @@ void _TestStableSortByKeyWithLargeKeysAndValues(void)
 
     for(size_t i = 0; i < n; i++)
     {
-        h_keys[i] = FixedVector<T,N>(rand());
-        h_vals[i] = FixedVector<T,N>(i);
+        h_keys[i] = FixedVector<T,N>(static_cast<T>(rand()));
```

Comment:
There is a random number facility in `unittest` that we should be using here instead.

Reply:
Done.